packages/alchemy/src/react/components/auth/card/index.tsx

```diff
@@ -48,7 +52,8 @@ export const AuthCard = (props: AuthCardProps) => {
         setAuthStep,
       }}
     >
-      <div className="modal-box flex flex-col items-center gap-5">
+      <div className="modal-box relative flex flex-col items-center gap-5 text-fg-primary">
+        <Navigation showingBack={false} onClose={closeAuthModal} />
```

Comment:
A follow-up will add the logic for the back button, which is hidden for now

Reply:
I don't think we should put this in the actual `AuthCard` impl, but in the modal. The AuthCard logic should work if a developer were to embed it as well, so if `useAuthModal` is present here and uiConfig is not (because they don't care about the modal) then this will error out

Reply:
Ah great catch, totally missed that